pennylane/_qubit_device.py

```diff
@@ -94,6 +94,7 @@ class QubitDevice(Device):
     _outer = staticmethod(np.outer)
     _diag = staticmethod(np.diag)
     _real = staticmethod(np.real)
+    _delete = staticmethod(np.delete)
```

Comment:
Was this used eventually?

Reply:
No it was for a first version of the partial trace, I'll remove it.